INTEROP.md

```diff
@@ -168,9 +168,38 @@ excludeDependentModules = true
 When both `excludeDependentModules` and `headerFilter` are used, they are
 applied as an intersection.
 
+### C compiler and linker options ###
+
+ Options passed to the C compiler (used to analyze headers, such as preprocessor definitions) and the linker
+(used to link final executables) can be passed in in the definition file as `compilerOpts` and `linkerOpts`
+respectively. For example
+
+<div class="sample" markdown="1" theme="idea" mode="c">
+
+```c
+compilerOpts = -DFOO=bar
+linkerOpts = -lpng
+```
+
+</div>
+
+Target-specific options, only applicable to the certain target can be specified as well, such as
```

Comment:
It is not clear that any `.def` option can be target-specific, including e.g. `headers`. Is this miss intentional?

Reply:
Nope, will fix.